snet_cli/commands.py

```diff
@@ -449,9 +449,9 @@ def get_model(self):
             model_dir = self._getstring("dest_dir") or Path("~").expanduser().joinpath(".snet").joinpath("models").joinpath(model_hash)
             if not os.path.exists(model_dir):
                 os.makedirs(model_dir)
-                model_tar = ipfs_client.cat(model_hash)
-                with tarfile.open(fileobj=io.BytesIO(model_tar)) as f:
-                    f.extractall(model_dir)
+            model_tar = ipfs_client.cat(model_hash)
+            with tarfile.open(fileobj=io.BytesIO(model_tar)) as f:
+                f.extractall(model_dir)
```

Comment:
May be we should remove all previous data to clear space for new model. What do you think?

Reply:
It's a good idea, if you see the line 449 if "dest_dir" isn't provided the IPFS model hash is concatenated with ".snet/models" to separate different .proto from services.
Maybe a better idea is to do this with "dest_dir", and then remove a previous IPFS model hash before get the new one.
But I'll have to analyse the impact of it when publishing/updating the service.

Reply:
Thanks - I didn't pay attention at line 449. I think is ok for now then.

Reply:
For now, I think we must not remove files/folder with this command.

Reply:
But probably not in case when ```dest_dir``` is omitted.